tools/cell_census_builder/util.py

```diff
@@ -10,11 +10,13 @@
 from scipy import sparse
 
 
-def array_chunker(arr: Union[npt.NDArray[Any], sparse.spmatrix]) -> sparse.coo_matrix:
+def array_chunker(
+    arr: Union[npt.NDArray[Any], sparse.spmatrix],
+    nnz_chunk_size: Optional[int] = 256 * 1024**2,  # goal (~2.4GiB for a 32-bit COO)
+) -> Iterator[sparse.coo_matrix]:
     """
     Return the array as multiple chunks, each a coo_matrix.
```

Comment:
Add param comment, e.g.,

nnz_chunk_size:  approximate number of elements in each chunk. 

I would also add a note that chunking is done by row, and will *never* divide a row across two chunks

Reply:
oh, doh - you are absolutely correct.